source/D0323r5.bs

```diff
@@ -73,28 +76,30 @@ public:
     unexpected() = delete;
     constexpr explicit unexpected(const E&);
     constexpr explicit unexpected(E&&);
-    constexpr const E& value() const &;
-    constexpr E& value() &;
-    constexpr E&& value() &&;
-    constexpr E const&& value() const&&;
+    constexpr explicit& operator=(const E&);
+    constexpr explicit& operator=(E&&) noexcept(see below);
```

Comment:
You mean expected, not explicit, for the two above :-)

Reply:
No. This two lines must be removed.
Sorry